assets/javascripts/templates/pages/settings_tmpl.coffee

```diff
@@ -20,6 +19,21 @@ app.templates.settingsPage = (settings) -> """
     </div>
   </div>
 
+  <div class="_settings-fieldset">
+    <h2 class="_settings-legend">Font:</h2>
+    <div class="_settings-inputs">
+      <label class="_settings-label _setting-font-family">
+        <select name="font" id="_setting-font-family">
+          <option value="arial" #{if (settings.font == 'Arial' or not settings.font) then ' selected' else ''}>Arial (default)</option>
+          <option value="cascadia" #{if settings.font == 'Cascadia' then ' selected' else ''}>Cascadia</option>
+          <option value="fira-code" #{if settings.font == 'Fira Code' then ' selected' else ''}>Fira Code</option>
+          <option value="proxima-nova" #{if settings.font == 'Proxima Nova' then ' selected' else ''}>Proxima Nova</option>
+          <option value="verdana" #{if settings.font == 'Verdana' then ' selected' else ''}>Verdana</option>
```

Comment:
I don't get the selection of the fonts. Some seem to be targetting the user interface (such as Arial). Some seem to be targetting the code blocks (such as Fira Code)?